guides/keras_cv/semantic_segmentation_deeplab_v3_plus.py

```diff
@@ -46,20 +46,25 @@ class label such as person, bike, or background to each individual pixel of an
 """
 
 """shell
-!pip install -q keras-core
-!pip install -q git+https://github.com/keras-team/keras-cv.git
+pip install -Uq keras-cv
```

Comment:
I've been using

pip install -q --upgrade keras-cv
pip install -q --upgrade keras  # Upgrade to Keras 3.

No strong preference, just keeping this for consistency.

Reply:
I added this while rendering.